src/App.js

```diff
@@ -90,21 +99,58 @@ class Root extends React.Component {
             }
           }
         }
-        this.props.historyUpdate({'data': [wsData.history]});
+        wallet.newAddressHistory(wsData.history);
```

Comment:
Is `hathorLib.` prefix missing?

Reply:
No, actually I have the utils in the `hathorLib` and utils specific from the wallet. This one is the wallet utils. Do you think we should name it differently?

Reply:
It is funny that I misunderstood it several times. But I think we are good.

Reply:
I think you were used with the old code, if we stay confused with this we can change in the future